README.md

```diff
@@ -32,6 +32,7 @@ This project builds against a combination of .NET Standard, .NET Core, and .NET
 Prerequisites to build:
 
 * Install the [.NET Core SDK](https://dotnet.microsoft.com/download)
+* Install [node](https://nodejs.org/en/)
```

Comment:
`[Node.js](https://nodejs.org)` instead? (using their branding and being language agnostic)